### DISCUSSION TITLE
Runtime Export: maintain instances ?

### DISCUSSION THREAD

**Discussion by FrogsInSpace** [atteneder/glTFast#436] [Q&A] [answered]
Looks like the runtime export does not respect instances of the same source prefab 
So if i have 5 clones of one prefab in the scene, each one gets stored seperately in the glb if i add the clones
I guess gltf would support multiple instanced nodes referencing the same source geometry , correct ?

In other words, how would i be able to fix that, assuming it could somehow be fixed on my side ?

**Comment by atteneder** [comment, marked as answer]
Hi @FrogsInSpace ,

If a mesh that is referenced multiple times is serialized more than once (instead of referenced multiple times in the glTF as well), than that's a bug. Could you file an issue and provide a scene that reproduces the error?
